Examples/Physics_applications/laser_ion/CMakeLists.txt

```diff
@@ -11,6 +11,16 @@ add_warpx_test(
     OFF  # dependency
 )
 
+add_warpx_test(
+    test_2d_laser_ion_acc_time_avg  # name
+    2  # dims
+    2  # nprocs
+    inputs_test_2d_laser_ion_acc  # inputs; will not be used b/c dependent test
+    analysis_time_avg_diags.py  # analysis
+    diags/diagTimeAvg/  # output
+    test_2d_laser_ion_acc  # dependency
```

Comment:
I had hoped that it would not run the full WarpX simulation if I specified the dependency. But I guess that is not true. In principle, it is enough to run the analysis script on the data of the main test.
Is there a quick way to do this or should we just run this twice and write double the amount of data?

Reply:
I'm not sure if I understand this correctly, you are saying that the dependent test `test_2d_laser_ion_acc_time_avg` is not supposed to run WarpX but only an analysis script on the result produced by another test? In that case I would just add the new analysis to the existing analysis script that is already run for the other test. 

The dependency is typically used for restart tests or for preparatory scripts (e.g., LASY scripts that produce a LASY file), meaning that the dependent test always runs WarpX.

Reply:
I see. Yes, in principle, I just need to run a script on the data that already `test_2d_laser_ion_acc` produces.
I will just combine the normal regression test with the time averaged diagnostics test, then. :)

Reply:
I made the necessary changes.